docs/example-scenario/hpc/video-rendering.md

```diff
@@ -0,0 +1,179 @@
+---
+title: 3D video rendering on Azure
+description: Running native HPC workloads in Azure using the Azure Batch service
+author: adamboeglin
+ms.date: 07/13/2018
+---
+# 3D video rendering on Azure
+
+3D rendering is a time consuming process that requires a significant amount of CPU time co complete.  On a single machine, the process of generating a video file from static assets can take hours or even days depending on the length and complexity of the video you are producing.  Many companies will purchase either expensive high end desktop computers to perform these tasks, or invest in large render farms that they can submit jobs to.  However, by taking advantage of Azure Batch, that power is available to you when you need it and shuts itself down when you don't, all without any capital investment.
+
+Batch gives you a consistent management experience and job scheduling, whether you select Windows Server or Linux compute nodes. With Batch, you can use your existing Windows or Linux applications, including AutoDesk Maya and Blender, to run large-scale render jobs in Azure.
+
+## Related use cases
+
+Other potential use cases that would have a similar environment include:
+
+* 3D Modeling
+* Visual FX (VFX) Rendering
+* Video transcoding
+* Image processing, color correction, & resizing
+
+## Architecture
+
+![Architecture overview of the components involved in a Cloud Native HPC solution using Azure Batch][architecture]
+
+This solution covers the workflow when using Azure Batch, the data flows through the solution as follows:
+
+1. Upload input files and the applications to process those files to your Azure Storage account
+2. Create a Batch pool of compute nodes in your Batch account, a job to run the workload on the pool, and tasks in the job.
+3. Download input files and the applications to Batch
+4. Monitor task execution
+5. Upload task output
+6. Download output files
+
+To simplify this process, you could also use the [Batch Plugins for Maya & 3ds Max][batch-plugins]
+
+## Deploy this scenario
+
+### Azure Batch: Creating an Azure Batch account and pools manually
+This sample solution is will provide help in learning how Azure Batch works while showcasing Azure Batch Labs as an example SaaS solution that can be developed for your own customers:
+
+[Azure Batch Masterclass][batch-labs-masterclass]
+
+### Azure Batch: Deploying a sample solution using an ARM (Azure Resource Manager) template
```

Comment:
Remove Azure Batch: 

Switch it to read: Deploying the sample scenario using an Azure Resource Manager (ARM) template